test/support/fake-usb.js

```diff
@@ -742,6 +742,11 @@ function addAssetTracker(options) {
 	return addDevice(opts);
 }
 
+function addP2(options) {
+	const opts = Object.assign({}, options, { type: deviceConstants.p2.name });
```

Comment:
Little JS style thing: you can use the spread operator to combine objects more concisely than with `Object.assign`. No need to change it here, just sharing a trick.
```suggestion
	const opts = { ...options, type: deviceConstants.p2.name };
```